main.py

```diff
@@ -29,8 +29,11 @@ def get_args_parser():
                         help='gradient clipping max norm')
 
     # Model parameters
+    parser.add_argument('--num_classes', type=int, default=20,
+                        help="Number of classes in your dataset. Overridden by coco and coco_panoptic datasets")
```

Comment:
Rather than "number of classes in your dataset", maybe specify that:

- it is the `max_class_id`,
- or is it `max_class_id+1`? https://github.com/facebookresearch/detr/issues/108#issuecomment-650269223